patches/45410914 - Rock Band 3.patch.toml

```diff
@@ -11,3 +15,25 @@ hash = "88E6DC86853C2522" # default.xex
     [[patch.be32]]
         address = 0x8226045c
         value = 0x485b5ddc
+
+# This patch requires a build of Xenia Canary that supports writable_code_segments, and having it enabled in the config.
+# Commit 28f3eb6 or later of RB3Enhanced recommended, due to removing the DEMO check.
+
+[[patch]]
+	name = "Load RB3Enhanced.dll"
+	author = "InvoxiPlayGames"
+	is_enabled = true
+	
+	# Replaces a check in _start with a branch to LoadLibrary
+	[[patch.be32]]
+		address = 0x8283CD64
+		value = 0x3C6082C6 # lis r3, 0x82C64350@hi
+	[[patch.be32]]
+		address = 0x8283CD68
+		value = 0x60634350 # ori r3, r3, 0x82C64350@lo
+	[[patch.be32]]
+		address = 0x8283CD6C
+		value = 0x48007F0D # bl LoadLibrary
+	[[patch.string]]
+		address = 0x82C64350
+		value = "RB3Enhanced.dll"
```

Comment:
EOF and uppercase hex, needs fixing

Reply:
Apologies, but what does the EOF comment mean in this instance. Thank you

Reply:
ah, got it

Reply:
Not fixed 
![](https://user-images.githubusercontent.com/37698908/230427294-7902ce43-f66c-4b88-8489-f2a3878e031f.png)

Reply:
my bad. now actually fixed